public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -112,48 +211,68 @@ code-example(language="sh" class="code-shell").
 code-example(language="sh" class="code-shell").
   ./node_modules/.bin/ng-xi18n
 
+.l-sub-section
+  :marked
+    Windows users may have to quote the command: like this `"./node_modules/.bin/ng-xi18n"`
+
 :marked
   By default the tool generates a translation file named **`messages.xlf`** in the
   <a href="https://en.wikipedia.org/wiki/XLIFF" target="_blank">XML Localisation Interchange File Format (XLIFF, version 1.2)</a>.
 
-code-example(language="sh" class="code-shell").
-  ./node_modules/.bin/ng-xi18n  --i18nFormat=xmb
-
-.l-sub-section
-  :marked
-    Windows users may have to quote the command:
-  code-example(language="sh").
-    "./node_modules/.bin/ng-xi18n"
-  :marked
-    Consider adding a convenience shortcut to the `scripts` section of the `package.json`
-    to make the command easier to remember and run:
-  code-example(format='.').
-    "scripts": {
-      "i18n": "ng-xi18n", 
-      ...
-    }
-  :marked
-    Now you can enter:
-  code-example(language="sh" class="code-shell").
-    npm run i18n
-
+a#other-formats
 :marked
   ### Other translation formats
 
   You can generate a file named **`messages.xmb`** in the 
   <a href="http://cldr.unicode.org/development/development-process/design-proposals/xmb"  target="_blank">XML Message Bundle (XMB)</a> format
-  by adding the `--i18nFormat=xmb` switch.
+  by adding the `--i18nFormat=xmb` flag.
+
+code-example(language="sh" class="code-shell").
+  ./node_modules/.bin/ng-xi18n  --i18nFormat=xmb
```

Comment:
why do we have 2 ws before --i18nFormat, I guess it's a placeholder for -p path/to/tsconfig ?